src/service/viewport/viewport-binding-ios-embed-sd.js

```diff
@@ -113,8 +113,9 @@ export class ViewportBindingIosEmbedShadowRoot_ {
       documentElement.classList.add('i-amphtml-body-minheight');
     }
 
-    const scroller = htmlFor(doc)`
-      <div id="i-amphtml-scroller">
+    const html = htmlFor(doc);
+    const scroller =
+      html`<div id="i-amphtml-scroller">
```

Comment:
bytes-saving idea: removing extra indentation whitespace. I don't think the linter will complain if you do something like this:

```js
const scroller = html`
<div id="i-amphtml-scroller">
<div id="i-amphtml-body-wrapper">
<slot></slot>
</div>
</div>
`;
```

Reply:
Ditto re: template whitespace.